### PR TITLE
Fix packer url

### PR DESCRIPTION
**Pull request by joshi4**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -6,4 +6,4 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.3.0'
 
-depends 'ark', '~> 0.9.0'
+depends 'ark', '~> 0.8.2'
```
Comment (sit) [inline metadata.rb]: What was the dep cycle?

Comment (joshi4) [inline metadata.rb]: sorry, this PR was against the wrong branch. was trying to fix it for our forked cookbook.  There seems to be another PR that fixes this: https://github.com/sit/chef-packer/pull/22 

teh dep cycle was caused by something else in our setup, not packer.
